### ISSUE TITLE
Import node/edge color, position and size from spreadsheets

### ISSUE THREAD

**Issue by TheG-Meister** [gephi/gephi#2775]
I've been using gephi in combo with excel for projects involving constructing manual networks. In order to enable the final part of my workflow I'm looking for a way to import the hidden values of node position, size and color from spreadsheet files such as .xlsx, .csv or .tsv.
### Proposed solution

I've heard importers for other file formats can read these attributes already. Simply from a UI perspective I was imagining that during the process of selecting data types for individual columns that these attributes could be selected there, but as best I've understood from browsing the repository I think all of those options are classes, while these attributes seem to be more specific. A more accessible option might be an additional checkbox in either the first or second import wizard that imports said columns only if they share the right name, a bit like the equivalent .csv export wizard that appends these columns in the first place.

### Alternatives considered

I haven't researched conversion options of raw text data to other formats such as .gdf that will import these attributes. In addition, I'm going to try updating the recast column plugin that allows already imported columns to be recast as these internal types.

**Comment by seinecle** [comment]
Hi!
For colors, the plugin "Give colors to your nodes" should do the trick? 
For positions, the geolayout plugin should be useful as well? 
For the size, any numerical attribute that you would call "size" would suffice, because then you dimension nodes according to it, et voilà ?

**Comment by mbastian** [comment]
Yes, the spreadsheet importer doesn't import those properties. @eduramiba is there any particular reason maybe that prevented us to implement this? I guess that was for version 0.9.2 when we touched this.

**Comment by roland779** [comment]
The node color feature would be really useful.
